### ISSUE TITLE
fix: 🤔 sd-teaser-media expandable slot not available via keyboard navigation

### ISSUE THREAD

**Issue by christophsaile** [solid-design-system/solid#1344]
## Current behavior
Currently the expandable slot in the sd-media-teaser does only react on hover.

![image](https://github.com/user-attachments/assets/6ab074e7-61ed-456f-98d8-dc318a1496ed)

As a keyboard user I am not able to open the expandable text. Furthermore this text is not rendered at all on small devices (as stated in the docs) which I think is not the best solution.

## Expected behavior
The Teaser has some icon-button (arrow down?) that indicates that I can expand the content. This icon-button can receive keyboard focus so that I can open and close the expandable text.

teaser with link and button see: https://github.com/solid-design-system/solid/issues/1311#issuecomment-2307097572

## Steps to reproduce
https://solid-design-system.fe.union-investment.de/x.x.x/storybook/?path=/story/components-sd-teaser-media--samples

## DoR
- [ ] Item has business value
- [ ] Item has been estimated by the team
- [ ] Item is clear and well-defined
- [ ] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by christophsaile** [comment]
@EllaELSA, @coraliefeil, @yoezlem can we schedule a meeting to check how we proceed with this issue ?:) were there already discussions about this CMS design?

**Comment by coraliefeil** [comment]
@smfonseca is this issue obsoltete after implementing https://github.com/solid-design-system/solid/issues/1491 ?

**Comment by smfonseca** [comment]
@coraliefeil yes, it is. We agreed and extended the accessibility hint with the information that the sd-expandable is only for mouse users and therefore should not contain important information in it.

**Comment by coraliefeil** [comment]
I’ve updated the description according to our alignment.

We need a design and dev ticket for this task, pls. @karlbaumhauer
